docs/components/autocomplete.md

```
@@ -0,0 +1,401 @@
+  <div class="space-y-4">
+    <fwb-autocomplete
+      v-model="selected1"
+      :options="countries"
+      :search-fields="['name']"
+      display="name"
+      label="Small"
+      size="sm"
+      placeholder="Small autocomplete..."
+    />
+    
+    <fwb-autocomplete
```

markdownlint-cli2 (0.17.2)

Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)